client/my-sites/plan-features/index.jsx

```diff
@@ -88,27 +93,31 @@ PlanFeatures.propTypes = {
 	available: PropTypes.bool,
 	onUpgradeClick: PropTypes.func,
 	// either you specify the plan prop or isPlaceholder prop
-	plan: React.PropTypes.string,
-	isPlaceholder: PropTypes.bool
+	plan: PropTypes.string,
+	isPlaceholder: PropTypes.bool,
+	isInSignup: PropTypes.bool
```

Comment:
Oh, I see. So, in Calypso UI `my-sites`, that are the site specific plans and in the signup flow that are the general plans.

Okay, nice idea, will do that. :+1: